src/addon-manager.js

```diff
@@ -462,6 +513,15 @@ class AddonManager extends EventEmitter {
       }
     }
 
+    // Verify that important fields exist in the manifest
+    let err = this.validateManifest(manifest);
+    if (err) {
+      console.error(
+        `Error found in manifest for ${packageName}\n${err}`);
```

Comment:
nit: pull the error string into a constant instead of typing it twice

Reply:
That's probably because I added the console.error intending to remove it before I checked it in. I'll double check.